modules/apps/foundation/frontend-theme/justin-theme/src/js/main.js

```diff
@@ -0,0 +1,75 @@
+AUI().ready(
+
+	/* This function gets loaded when all the HTML, not including the portlets, is loaded. */
+	hideHeadingScroll()
+);
+
+Liferay.Portlet.ready(
+
+	/*
+	This function gets loaded after each and every portlet on the page.
+
+	portletId: the current portlet's id
+	node: the Alloy Node object of the current portlet
+	*/
+	function(portletId, node) {
+	}
+);
+
+Liferay.on(
+	'allPortletsReady',
+
+	/* This function gets loaded when everything, including the portlets, is on the page. */
+	function() {
+	}
+);
+
+function hideHeadingScroll() {
+	var delta = 40;
+	var lastScrollTop = 0;
+	var navbarHeight = $('#banner').outerHeight();
+	var scrolling;
+
+	$(window).scroll(
+		function(event) {
+			scrolling = true;
+		}
+	);
+
+	setInterval(
+		function() {
+				if (scrolling) {
+					moveBar();
+					scrolling = false;
+				}
+		},
+		250
+	);
+
+	function moveBar() {
+		var scrollTop = $(this).scrollTop();
+
+		if (Math.abs(lastScrollTop - scrollTop) > delta) {
+
```

Comment:
No line break needed for spacing between ```if``` statements and it's subsequent statement.